src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -755,6 +757,13 @@ type PandaproxyAPITLS struct {
 	// duplicate the secret to the same namespace as redpanda CRD to be able to
 	// mount it to the nodes
 	NodeSecretRef *corev1.ObjectReference `json:"nodeSecretRef,omitempty"`
+	// If provided and RequireClientAuth is true, the operator uses the certificate
+	// in this secret instead of issuing client certificates. The secret is expected to provide
+	// the following keys: 'ca.crt'.
+	// If ClientCACertRef points to a secret in a different namespace, the operator will
```

Comment:
Do we have to have this ability to reference secrets in different namespaces? We're trying to work on reducing the security surface and this will set up precedent that we will get stuck following.

Reply:
I don't think it is must-have. I don't see big benefit.  @0x5d What you do think? Use LocalObjectReference instead?

Reply:
Sounds good! I was following the established pattern, but it makes sense to move away from it.